augur/8.0.0/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM ubuntu:bionic
+
+# metadata
+LABEL base.image="ubuntu:bionic"
+LABEL container.version="1"
```

Comment:
Please change to `LABEL dockerfile.version="1"`. See #118 for why

Reply:
Patched in `/data` and `dockerfile.version`. Thanks for the review! 

I've been meaning to make a checklist of all the little things that are easy to forget (locale, /data, update Program_Licenses.md, etc.)